kola/harness.go

```diff
@@ -409,6 +409,11 @@ func runTest(h *harness.H, t *register.Test, pltfrm string) {
 				h.Errorf("Found %s on machine %s console", badness, id)
 			}
 		}
+		for id, output := range c.JournalOutput() {
+			for _, badness := range CheckConsole([]byte(output), t) {
+				h.Errorf("Found %s on machine %s journal", badness, id)
```

Comment:
For messages that go both places, this will produce redundant log entries.  That seems okay as a starting point, though we might want to split out some checks later.

Reply:
Yeah, in the future I'd like to switch from using the `CheckConsole` function for the journal checks to a more specific set.